echidna/utilities.py

```diff
@@ -140,6 +140,8 @@ def start_logging(short_name=False, script_name=True):
 
     # Set up logging to file
     path = output.__default_save_path__ + "/"
+    if os.path.isdir(path+"output/logs/"):
+        path += "output/logs/"
```

Comment:
I'm wondering if it would be cleaner to add `echidna.__base_dir__ + "/output/logs/"` as the default save path in the [`user_profile`](https://github.com/snoplusuk/echidna/blob/master/echidna/settings.py#L25) and then just set `path` as that.